app/models/customer.rb

```diff
@@ -0,0 +1,20 @@
+class Customer < ApplicationRecord
+  has_many :rentals, dependent: :destroy
+  validates :name, presence: true
+  validates :registered_at, presence: true
+  validates :address, presence: true
+  validates :city, presence: true
+  validates :state, presence: true
+  validates :postal_code, presence: true
+  validates :phone, presence: true
+
+  def get_count
+    items_checked_out = self.rentals.select { |rental| rental.check_in_status == false }
+
+    if !items_checked_out.empty?
```

Comment:
Why not just return `items_checked_out.length` or `.count` why have the if statement?